.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -21,11 +21,3 @@
 **Removed**
 
 - {{removed thing}}
-
-<!-- Deploy Previews are enabled by applying the following labels for the corresponding package: -->
-<!-- *** "package: react" -->
-<!-- *** "package: patterns" -->
-<!-- *** "package: vanilla" -->
-<!-- *** "package: services" -->
-<!-- *** "package: utilities" -->
-<!-- *** "package: styles" -->
```

Comment:
@mkothur this might have been a bad merge, looks like changes from a day or so ago are gone...